src/Symfony/Component/Validator/Resources/translations/validators.pt_BR.xlf

```diff
@@ -192,7 +192,7 @@
             </trans-unit>
             <trans-unit id="51" resname="No temporary folder was configured in php.ini.">
                 <source>No temporary folder was configured in php.ini, or the configured folder does not exist.</source>
-                <target state="needs-review-translation">Não foi configurada uma pasta temporária no php.ini, ou a pasta configurada não existe.</target>
+                <target>Nenhum diretório temporário foi configurado no php.ini, ou o diretório configurado não existe.</target>
```

Comment:
The change here is intended?

Reply:
@xabbuh Yes, if it's an issue, I can revert this change. Upon reviewing it now, I think I might have made the change unnecessarily. 

The intention was to make the sentence a bit more technical by replacing the word "pasta" (folder in english) with "diretório" (directory in english) and changing the negation and subject of the sentence to the beginning, making the sentence follow the subject-verb-object linguistic typology.